api/google-meets/documentation/1.0.0/google-meets.json

```diff
@@ -404,6 +402,15 @@
               "newMeetingTimes": {
                 "type": "component"
               },
+              "opportunities": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              "isPlatform": {
```

Comment:
Why do we care if the google-meets is platform or independent product?

Reply:
Oh this is documentation of every endpoint.

Reply:
Yeah these documentations are actually automated.